Framework/DataHandling/src/Load.cpp

```diff
@@ -340,6 +340,9 @@ void Load::exec() {
 
   // Set the remaining properties of the loader
   setOutputProperties(m_loader);
+  // Set the loader and version from correct loader
+  setPropertyValue("LoaderName", m_loader->name());
```

Comment:
This should be protected with an `if (m_loader)` in case no loader was chosen. The `assert` in `getFileLoader()` is only run in debug builds.

Reply:
Thanks, I added this protection and a lengthier comment explaining the reason for those two lines.